### DISCUSSION TITLE
How to analyze a core dump with nvim-dap?

### DISCUSSION THREAD

**Discussion by shmerl** [mfussenegger/nvim-dap#1320] [Q&A]
Is there a way to analyze a core file with nvim-dap using gdb somehow or such scenario is impossible?

**Comment by DavidePatria** [comment]
did you find a way?

**Reply by shmerl** [reply]
Nope, I don't think it's possible unless I misunderstand something. I.e. it's not something DAP protocol itself supports. So you'd just need to use gdb directly.
